lib/global_id.rb

```diff
@@ -1,9 +1,14 @@
 require 'global_id/global_id'
+require 'active_support'
 
 autoload :SignedGlobalID, 'global_id/signed_global_id'
```

Comment:
Not really. We can implement `self.eager_load!` in `GlobalID` and make it require that file.